src/openms/source/DATASTRUCTURES/SuffixArraySeqan.cpp

```diff
@@ -171,7 +171,7 @@ namespace OpenMS
   //destructor
   SuffixArraySeqan::~SuffixArraySeqan()
   {
-
+    delete it_;
```

Comment:
and document what is wrong with it...maybe we could call delete on the old object before creating a new one?